cmd/deploy/deploy.go

```diff
@@ -60,15 +61,16 @@ var tempKubeConfigTemplate = "%s/.okteto/kubeconfig-%s-%d"
 
 // Options options for deploy command
 type Options struct {
-	ManifestPath     string
-	Name             string
-	Namespace        string
-	K8sContext       string
-	Variables        []string
-	Manifest         *model.Manifest
-	Build            bool
-	Dependencies     bool
-	servicesToDeploy []string
+	OriginalManifestPath string
+	ManifestPath         string
```

Comment:
I would include a comment to specify the difference between these 2 paths. Our future selves will thank us for that (same for other places where we define these 2 properties)